synapse/lib/stormlib/aha.py

```diff
@@ -85,6 +111,73 @@ async def _methAhaGet(self, svcname, filters=None):
         proxy = await self.runt.snap.core.reqAhaProxy()
         return await proxy.getAhaSvc(svcname, filters=filters)
 
+    async def _methCallPeerApi(self, svcname, apiname, timeout=None, skiprun=None):
+        '''
+        Call an API on an AHA service.
+
+        Args:
+            svcname (str): The name of the AHA service to call
+            apiname (str, tuple): The API name or todo tuple (name, args, kwargs)
```

Comment:
We prefer not to overload types this way. Requiring the caller to construct a todo tuple ( or maybe plumbing a helper for easily building one? ) isn't too much of a burden vs the complexity for such a powerful API.